bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java

```diff
@@ -267,6 +282,31 @@ else if (String.class.isAssignableFrom(type))
         }
     }
 
+    private String promptForListItem(String message, Collection<String> items, String defaultValue)
```

Comment:
This seems like it should be in a utility somewhere. Maybe we could move these to static functions in Bootstrap.java?